docker/quickstart/README.md

```diff
@@ -1,6 +1,10 @@
 # DataHub Quickstart
 To start all Docker containers at once, please run below command:
 ```
+export DATA_STORAGE_FOLDER=/tmp/datahub
+mkdir -p ${DATA_STORAGE_FOLDER}
+mkdir -p ${DATA_STORAGE_FOLDER}/elasticsearch
+chown 1000:1000 ${DATA_STORAGE_FOLDER}/elasticsearch
```

Comment:
Why do you need to create `elasticsearch` folder and play with its access controls separetely?

Reply:
The elasticsearch docker is trying to create some folders inside the volume without having creation rights -- https://discuss.elastic.co/t/elastic-elasticsearch-docker-not-assigning-permissions-to-data-directory-on-run/65812/4

Reply:
I see, thanks for the reference. So, as I understand (from a little bit of reading on the topic), this configuration is needed because you use a `bind mount` (or `host mounted data volume`). Why do you prefer that over data volumes which seems like the preferred mechanism from Docker documentation? https://docs.docker.com/storage/volumes/

Reply:
The difference between them it's that if you do a `docker-compose down` you will kill and lose the volumes data. One approach is more docker friendly and the other one is more "user-friendly".

Reply:
That sounds good but this needs to be documented clearly. Can you create a `quickstart.sh` which contains all these commands and update the instructions in main README to run this script?